.github/actions/build-zip-lint/action.yml

```
@@ -0,0 +1,55 @@
+        dotnet publish .\src\Community.PowerToys.Run.Plugin.CrcLauncher.csproj -c Release /p:PublishProfile=${{ inputs.architecture }}
+
+    - name: Clean and Zip the build output
+      id: zip-output
+      shell: pwsh
+      run: |
+        $zipPath = "./${{ env.ZipPrefix }}-${{ inputs.version }}-${{ inputs.architecture }}.zip"
+        Remove-Item -Recurse "./src/bin/${{ inputs.architecture }}/Release/${{ env.FrameworkVersion }}/${{ env.BuildFolderName }}/runtimes"
+        Remove-Item "./src/bin/${{ inputs.architecture }}/Release/${{ env.FrameworkVersion }}/${{ env.BuildFolderName }}/PowerToys.*"
+        Remove-Item "./src/bin/${{ inputs.architecture }}/Release/${{ env.FrameworkVersion }}/${{ env.BuildFolderName }}/Wox.*"
+        Remove-Item "./src/bin/${{ inputs.architecture }}/Release/${{ env.FrameworkVersion }}/${{ env.BuildFolderName }}/*.xml"
+        Remove-Item "./src/bin/${{ inputs.architecture }}/Release/${{ env.FrameworkVersion }}/${{ env.BuildFolderName }}/Microsoft.Windows.SDK.NET.dll"   
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)